### ISSUE TITLE
Typo in Cluster Documentation

### ISSUE THREAD

**Issue by bring52405** [galaxyproject/galaxy#7433]
```
<plugins>
    <plugin id="cli" type="runner" load="galaxy.jobs.runners.cli:CLIJobRunner"/>
</plugins>
```
should actually be:
```
<plugins>
    <plugin id="cli" type="runner" load="galaxy.jobs.runners.cli:ShellJobRunner"/>
</plugins>
```
Got the correct syntax from the job_conf.xml.sample_advanced

**Comment by martenson** [comment]
@bring52405 would you mind opening a pull request?

**Comment by hexylena** [comment]
Fixed in #7486
